src/entries/EntryName.jsx

```diff
@@ -0,0 +1,43 @@
+function EntryName(entry, nameType="short", includeVersion=false) {
```

Comment:
Since this is just a plain JS file, you can use just `.js` extension.

Reply:
done.